### DISCUSSION TITLE
Incendiary grenade destroys the detonator. Acid grenade does not destroy the detonator. Was this intended?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15221] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

IMHO Incendiary grenade should not destroy the detonator as well as acid grenade.

### Reproduction steps

1. Insert Incendiary grenade in the detonator. Detonate it.
2. Detonator has gone.
3. Insert acid grenade in the detonator. Detonate it.
4. Detonator left unharmed.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
In my opinion this is behaving as expected.
A grenade containing incendium bursting into high intensity flames destroying a detonator seems logical to me.
A grenade exhuming acid gasses could also damage/destroy a detonator in theory, but is a lot less violent than the Incendiary Grenade.  

XML has itemdamage defined as 200 for the Incendiary Grenade and 0 for the Acid Grenade so nothing is bugged here, closing.
It would have been more fitting to submit this as a suggestion, rather than bug.
